plugins/catalog-backend/src/stitching/Stitcher.test.ts

```diff
@@ -110,6 +112,15 @@ describe('Stitcher', () => {
       });
 
       expect(entity.metadata.etag).toEqual(entities[0].hash);
+      const last_updated_at = entities[0].last_updated_at;
+      expect(last_updated_at).not.toBeNull();
+      const lastUpdatedAt = DateTime.fromMillis(
+        last_updated_at ? +last_updated_at : 0,
+      );
+      const msAfterStitch = lastUpdatedAt
+        .diff(timeBeforeStitch, 'milliseconds')
+        .toObject();
```

Comment:
You should use `.as('milliseconds')` on Duration, otherwise you might get `{ hours: 1, milliseconds: 0 }` :) And maybe check that it's smaller than 10000 or whatever (basically that it's somewhere near the expected value; that's basically all we can do). Some in the similar thing farther down.

may have to use abs() on it too if there's a small diff between the db and the jest runner 🤔 

Come to think of it ... some of these tests are run against a testcontainers wrapped database and I'm not sure there's even any sensible way of testing what the value of this column is. Only safe way would be to compute the time diff in an SQL statement (staying within server time) and checking the output. Not worth it most likely. Could honestly skip this test I think

Reply:
thanks for advice. It's changed to expect the value of last_updated_at to be not null.